RedHat-UX/red-hat-design-system
# docs: section spacing

## What I did

1. Removed section shortcodes and cleaned up extra spacing from docs pages.
2. Added `margin-top: 80px` before h2 elements in `_base.scss`
3. Removed margins from empty paragraph elements in `_base.scss`

## Testing Instructions

1. Review docs pages in [DP](https://deploy-preview-1055--red-hat-design-system.netlify.app/).
2. Make sure spacing above h2 elements is 80px.
3. Make sure that find & replace did not break any page.

## Notes to Reviewers
